nomt/src/rw_pass_cell/mod.rs

```diff
@@ -383,20 +394,34 @@ impl<'a, 'pass, T> ReadGuard<'a, 'pass, T>
 where
     'pass: 'a,
 {
+    #[cfg(not(loom))]
     /// Get a reference to the underlying value.
     pub fn get(&self) -> &'pass T {
         // SAFETY: The existence of the guard ensures that there are only shared references
         //         to the inner value. The returned reference cannot outlive the guard.
         unsafe { &*self.inner.get() }
     }
+
+    #[cfg(loom)]
+    pub fn with<R>(&self, f: impl FnOnce(&T) -> R) -> R {
```

Comment:
since it's `cfg(loom)` only, does it need to be public?

Reply:
you're right, it shouldn't be needed